addons/test_mail/tests/test_message_track.py

```diff
@@ -353,6 +359,12 @@ def test_message_track_monetary(self):
             ('company_currency', 'many2one', self.user_employee.company_id.currency_id, self.company_2.currency_id)
         ])
 
+    def test_message_track_monetary_no_company(self):
+        try:
+            self.record_no_company.write({'amount': 300})
```

Comment:
Could be great to use ``assertTracking`` to check that it did not crash and that we actually tracked the value :)